### PR TITLE
Updated the documentation for the Timer Component #10846

### PR DESCRIPTION
**Pull request by krishnadev7**
#  Description

Closes [10846](https://github.com/ToolJet/ToolJet/issues/10846)

### Changes Made

1. Formatting updates:  
- Updated the formatting based on the latest guidlines 
- For each h2 (created using ##), removed 24px padding-bottom.

2. Content updates: 

-  Added more descriptive introduction
- Replaced the word "widget" with "component" everywhere
- Replaced the header "Layout" with "Devices"
- Updated the Description and Expected Value columns 
- Added a divider before the "Styles" section
- Updated the description in the Events table
- Updated the styles table with three headers - Property, Description, Configuration Options
- Updated the content of the style table
- Included box-shadow under the Styles table

## Files Modified

- Next -` ToolJet/docs/docs/widgets/timer.md`
- Version 2.50.0 (LTS) - `ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/timer.md`

### CODE REVIEW

**Comment by RudraPratik30** [comment]
Hey @krishnadev7!
Can you please correct the syntax on line no. 81?
<hr> tag must be written as <hr/> since it is a self-closing tag.
Please do it so that we can deploy this on render and complete the review.

File: docs/docs/widgets/timer.md [hunk 1]
```diff
@@ -2,90 +2,94 @@
...
 
 ## Properties
 
-| <div style={{ width:"100px"}}> Property </div>     | <div style={{ width:"100px"}}> Description </div> |
-|:----------- |:----------- | 
-| Default value | It specifies the initial value of timer. Format is: `HH.mm.ss.SS`. |
-| Timer type | It specifies If its a upward or downward counter. Select `Count Up` or `Count Down` from the dropdown or you can click on `Fx` to programmatically define the values `countUp` or `countDown`. |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div>                                                                                                                                              |
+| :--------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default value                                  | It specifies the initial value of timer. Format is: `HH.mm.ss.SS`.                                                                                                                             |
```
Comment (RudraPratik30) [inline docs/docs/widgets/timer.md:16]: Hey, the format should be `HH.MM.SS.MS`, can you please change this?

File: docs/docs/widgets/timer.md [hunk 2]
```diff
@@ -2,90 +2,94 @@
...
 
-| <div style={{ width:"100px"}}> Event </div>     | <div style={{ width:"100px"}}> Description </div> |
-|:----------- |:----------- | 
-| On start | This event is fired when user clicks on start button. |
-| On resume | This event is fired when user clicks on resume button. |
-| On pause | This event is fired when user clicks on pause button. |
-| On count down finish | This event is fired when the count down timer reaches zero. |
-| On reset | This event is fired when user clicks on reset button. |
+| <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div>                                                                 |
+| :------------------------------------------ | :---------------------------------------------------------------------------------------------------------------- |
```
Comment (RudraPratik30) [inline docs/docs/widgets/timer.md:26]: Hey @krishnadev7, can you please recheck this table?
The events and their descriptions do not match. Also, this is the timer component, you have added description for the text input component, please correct that too.
Thanks!

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

**Comment by krishnadev7** [comment]
Hey @RudraPratik30 , I accidentally closed the PR and can't reopen it. I've raised a new PR. Please take a look!

**Comment by RudraPratik30** [comment]
Hey @krishnadev7, there must be an option to reopen the PR, @PriteshKiri can you help us with this?

Also @krishnadev7 I can see you have opened 2 new PRs #10880 and #10881 can you close the one that is not needed?

**Comment by PriteshKiri** [comment]
Hey @RudraPratik30, we can keep this issue closed for now.
@krishnadev7, please close the stale PR and let us know which one needs to be reviewed.

**Comment by krishnadev7** [comment]
hey @PriteshKiri  Closed PR #10880. I would like to review PR #10881. Thanks!

**Pull request closed without merging**